src/playground/server.clj

```diff
@@ -22,6 +22,23 @@
       server/default-interceptors
       server/dev-interceptors))
 
+(def test-map
+  (-> (service/service 59800) ;; TEST configuration
```

Comment:
Don't have the 59800 scattered around

In general develop a sensibility against duplication. In case of doubt, prefer DRY over duplication